feat: Arrays should be indexable from the end too (#3004)
confluentinc/ksql: docs/developer-guide/query-with-arrays-and-maps.rst

Press Ctrl+C to terminate the query.

> Arrays elements can be accessible using positive or negative index.

Reviewer: ```suggestion
You can access array elements by using positive or negative index values.
```

> For instance, to get the user last interest run the following select:

Reviewer: ```suggestion
For example, to get the user's last interest run the following SELECT statement:
```
